### PR TITLE
Meeting minutes, 2020-04-23

### PR DESCRIPTION
**Pull request by graeme-a-stewart**
Additional small correction to 2020-04-09

### CODE REVIEW

File: organization/_posts/2020/2020-04-23-coordination.md [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+
+- Could do more than an LoI as we have made progress (LHCC docs)
+- What length? Exec summary, that then references longer document(s)
+- Should continue discussion in the coordination group, inviting Oli (who is a Computational Frontier convenor)
+
+## Google Summer of Code 2020
+
+GSoC Advisory Committee this year is Liz, Pere and Michel.
+
+This year we have ~200 student proposals for 47 project ideas, 37 qualifying (valid student proposal, complying with the program rules). We asked for 36-37 slots and **received 36** (last year we had 33). The successful proposals will be announced on May 4, coding starts June 1st.
```
Comment (amete) [inline organization/_posts/2020/2020-04-23-coordination.md:27]: Perhaps "student proposal" => "student proposals"

File: organization/_posts/2020/2020-04-23-coordination.md [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+
+- We are a bit long, currently:
+    - Intro 2
+    - Generators 5
+    - Simulation 7
+    - Reconstruction 6
+    - Analysis 6
+    - Conclusions 2
+    - **Total 29**
+- LaTeX should allow a squeeze (technical "solution"), but we should look out for opporunities to be more concise
```
Comment (amete) [inline organization/_posts/2020/2020-04-23-coordination.md:85]: opporunities => opportunities

File: organization/_posts/2020/2020-04-23-coordination.md [hunk 3]
```diff
@@ -0,0 +1,210 @@
...
+
+### Introduction
+
+- Reminder that we align expected hardware gains with WLCG/ATLAS/CMS
+    - This was checked and is estimated at ~10% per year as the common figure to be used (the experients also give a 20% pa figure, but that seems over-optimistic)
```
Comment (amete) [inline organization/_posts/2020/2020-04-23-coordination.md:90]: the _experients_ also give a 20% _pa_ figure => the **experiments** also give a 20% **per year** figure?

File: organization/_posts/2020/2020-04-23-coordination.md [hunk 4]
```diff
@@ -0,0 +1,210 @@
...
+
+### Detector Simulation
+
+- Some (minimal) discussion on conclusions wording needed
+- References will be fixed on Overleaf
+    - We need to find a few references (for the moment just mentioned, but no details written down)
+
+### Reconstruction and Software Triggers
+
+- DL: There are some overlapping ideas to consider consolidate with respect to the facilities document to LHCC (perhaps more general than the reco/trigger secion).
```
Comment (amete) [inline organization/_posts/2020/2020-04-23-coordination.md:104]: secion => section

File: organization/_posts/2020/2020-04-23-coordination.md [hunk 5]
```diff
@@ -0,0 +1,210 @@
...
+- One idea we didn't capture so well is the need to support HPC architectures. This comes out in both validation and non-x86 support. Will add a few sentences today.
+ 
+### Data Analysis
+
+- Improved the coherence of the document since last  week, we added several important points for ~the same word count but could maybe be more concise.  Some references need to be found/fixed.
+- We could still be a little more forthright in the R&D needs section and point out what we think useful directions are. For example, pointing out how beneficial it has been to open up to new ways of thinking (DataFrames et al), this healthy competition philosophy should continue.
+
+### ~~Conclusions~~ Summary
+
+- *Summary* is a more approriate way to finish the document
```
Comment (amete) [inline organization/_posts/2020/2020-04-23-coordination.md:114]: approriate => appropriate

File: organization/_posts/2020/2020-04-23-coordination.md [hunk 6]
```diff
@@ -0,0 +1,210 @@
...
+
+No meeting in May due to workshop, but some (incl. categories) potentially lined up after interactions for LHCC doc: 
+
+- Accelerators in trigger & reconstruction (Allen/Patatrack updates since CHEP + contribution from ATLAS, topics can be chosen following HSF workshop contributions)
+- [category of meetings] How experiments sharing pieces of LHC experiments can reuse LHC trigger & reco code. Cases in mind: FASER, LDMX (but maybe more, e.g. future colliders?)
+- CBM experiment and 4D (time-slice) tracking reconstruction
+- FASTML (HSF4ML & SONIC) for ML on accelerators
+- (from earlier) Particle flow review
+
+Would it be worth categorizing meetings according to priorities in LHCC document? May benefit HSF PR...
```
Comment (amete) [inline organization/_posts/2020/2020-04-23-coordination.md:153]: in **the** LHCC document

File: organization/_posts/2020/2020-04-23-coordination.md [hunk 7]
```diff
@@ -0,0 +1,210 @@
...
+
+*Using the LHCC document to plan the next activities for the working groups is strongly supported.*
+
+#### Other
+
+We have a request of someone who wanted to work in trigger & reco (see previous meetings).
```
Comment (amete) [inline organization/_posts/2020/2020-04-23-coordination.md:159]: We have a request _of_ someone who _wanted_ to work _in_ => We have a request **from** someone who **wants** to work **on**?

File: organization/_posts/2020/2020-04-23-coordination.md [hunk 8]
```diff
@@ -0,0 +1,210 @@
...
+
+## News, general matters
+
+[Snowmass process](https://snowmass21.org), is there scope for a short HSF submission? (e.g. similar to the [paper](https://zenodo.org/record/2413005) we sent to EPPSU)
+
+CD's opinion (1st time snowmass-er, others may have further experience): LoI + paper combination welcome (there is a Computational Frontier). Instructions in links here: <https://snowmass21.org/loi>.
+Paper encourages projects to develop further within the framework with "help" of a forum (at least that's how things are working in Energy Frontier). 
+
+- Could do more than an LoI as we have made progress (LHCC docs)
+- What length? Exec summary, that then references longer document(s)
```
Comment (eduardo-rodrigues) [inline organization/_posts/2020/2020-04-23-coordination.md:20]:
```suggestion
- What length? Exec summary, then references to longer document(s)
```

File: organization/_posts/2020/2020-04-23-coordination.md (hunk 3 above)
Comment (graeme-a-stewart) [reply organization/_posts/2020/2020-04-23-coordination.md:90]: *pa* is *per annum*, so it's correct, but maybe not so clear as if spelled out

**Comment by graeme-a-stewart** [comment]
Thanks both for the ready through and corrections!

**Pull request merged**